File: BOJ_2023/10808_알파벳_개수/main.cpp
// https://www.acmicpc.net/problem/10808

#include <bits/stdc++.h>
using namespace std;
char str[101];
int arr[26];

int main() {
  cin >> str;
  for (int i = 0; i < strlen(str); i++) {
    arr[str[i] - 'a'] += 1;
  }
  for (auto i : arr) cout << i << " ";
  return 0;
}

File: BOJ_2023/10808_알파벳_개수/main.js
// https://www.acmicpc.net/problem/10808

const readline = require('readline')
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
})

rl.on('line', (line) => {
  solve(line)
  process.exit()
}).on('close', () => {
  process.exit()
})

function solve(input) {
  const answer = new Array(26).fill(0)
  for (let i = 0; i < input.length; i++) {
    answer[input.charCodeAt(i) - 'a'.charCodeAt(0)]++
  }
  console.log(answer.join(' '))
}
